ui/StatusQ/src/StatusQ/Components/StatusChatList.qml

```diff
@@ -114,7 +111,7 @@ Item {
                     id: draggable
                     objectName: model.name
                     width: root.width
-                    height: model.categoryOpened ? statusChatListItem.height : 0
+                    height: model.categoryOpened ? statusChatListItem.height + 4 /*spacing between non-collapsed items*/ : 0
                     visible: height
                     verticalPadding: 2
```

Comment:
What do you mean? It works and fixes the reported issue

Reply:
I mean I tested locally to add to `verticalPadding` to see if it did anything and it didn't, so I opened the issue.

I thought the `+ 4` you did above was the thing that did the trick, but maybe I'm wrong and the `verticalPadding` is still needed.

Reply:
Give this PR a try ;)

> I mean I tested locally to add to `verticalPadding` to see if it did anything and it didn't, so I opened the issue.

There's no `verticalPadding`, our delegate is a `StatusListItem` (is a `Rectangle`) and has no such thing...

Ah, it's a `Control` now... gotta do it differently then :)

Reply:
@jrainville if this PR works for you, good to merge then; if not, I'll have a second look tomorrow :)

Reply:
I tested the PR, it works fine. I even removed `verticalPadding: 2` and it still works :sweat_smile:

Reply:
Yeah it's because the delegate is now a Control which takes its implicitHeight from its contentItem; which doesn't work with a MouseArea. My approach should work in any case w/o major headaches with refactoring the whole thing 😊